gnocchi/tests/functional/gabbits/aggregates.yaml

```diff
@@ -0,0 +1,212 @@
+fixtures:
```

Comment:
it'd be great to have one scenario for fetch and one for search-and-fetch

Reply:
done